test/e2e_new/data/e2e_conf.yaml

```diff
@@ -88,18 +88,29 @@ providers:
             new: capa-manager:e2e
     files:
       # Add a cluster template
-      - sourcePath: "../../templates/cluster-template.yaml"
+      - sourcePath: "../../../templates/cluster-template.yaml"
+      - sourcePath: "../../../_artifacts/templates/cluster-template-conformance-ci-artifacts.yaml"
 
 variables:
   KUBERNETES_VERSION: "v1.18.2"
   CNI: "./data/cni/calico.yaml"
   AWS_CONTROL_PLANE_MACHINE_TYPE: t3.large
   AWS_NODE_MACHINE_TYPE: t3.large
+  CONFORMANCE_CI_ARTIFACTS_KUBERNETES_VERSION: "v1.18.2"
+  CONFORMANCE_WORKER_MACHINE_COUNT: "3"
+  CONFORMANCE_CONTROL_PLANE_MACHINE_COUNT: "1"
```

Comment:
What numbers should we use here. Chose 1 CP machine for expediency in getting the cluster up.